tasks/docker.yml

```diff
@@ -43,7 +43,7 @@
 
 - name: Ensure pi user is added to the docker group.
```

Comment:
Can you also change this to `{{ ansible_user }}` (where it says `pi`).

Reply:
removed the directory issue and updated the task name to include the var as well.